.github/workflows/test_workflow_scripts/golang-http-linux.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+source ./../../.github/workflows/test_workflow_scripts/test-iid.sh
+echo "iid.sh executed"
+
+# Checkout a different branch
+git fetch origin
+#git checkout native-linux
+
+# Check if there is a keploy-config file, if there is, delete it.
+if [ -f "./keploy.yml" ]; then
+    rm ./keploy.yml
+fi
+
+rm -rf keploy/
+
+# Build go binary
+CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o http-pokeapi
```

Comment:
Why did you mention CGO_ENABLED, GOOS, GOARCH, when it is the default values for the current pipeline?